akka-stream-typed/src/main/scala/akka/typed/stream/scaladsl/TypedSink.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com/>
+ */
+
+package akka.typed.stream.scaladsl
+
+import akka.stream.scaladsl._
+import akka.typed._
+import akka.NotUsed
+
+object TypedSink {
```

Comment:
not sure `Typed` should be in the name

Reply:
Yeah... I'm open to better names, just saying `Sink` will then easily cause conflicts in Java DSL, because there people will import that class and call `ThingSink.thingy`.  In Scala it's fine, people would just get the conversions onto the existing types, so it could be Sink or just the package or something...

Open for naming ideas here! :)